release notes/v0.53.0.md

```diff
@@ -0,0 +1,266 @@
+k6 `v0.53.0` is here 🎉! This release includes:
+
+- Native ECMAScript modules support :tada:!
+- New experimental OpenTelemetry metrics output :rocket:!
+- Blob support in experimental websockets module
+- Consolidating cloud features and commands under `k6 cloud` :cloud:
+- Breaking change: remove magic URL resolutions :stop_sign:
+
+## Breaking changes
+
+### Require is now specification compliant and always resolve based on the file it is written in [#3534](https://github.com/grafana/k6/issues/3534)
+
+`require` in k6 used to resolve identifiers based on the current "root of execution" (more on that later). In a lot of cases that aligns with the file the `require` is written in or a file in the same folder, which leads to the same result. In a small subset of cases, this isn't the case.
```

Comment:
```suggestion
The `require` keyword in k6 used to resolve identifiers based on the current "root of execution" (more on that later). In a lot of cases that aligns with the file the `require` is written in or a file in the same folder, which leads to the same result. In a small subset of cases, this isn't the case.
```

Reply:
require is not keyword - it is just a function

Reply:
Can we change it to "The `require` function"?